util/types.go

```diff
@@ -27,4 +27,9 @@ var (
 		"CAP_SETUID",
 		"CAP_SYS_CHROOT",
 	}
+	// DefaultNetworkSysctl is the list of Kernel parameters which we
+	// grant by default to containers which are running under UID 0.
+	DefaultNetworkSysctl = map[string]string{
+		"net.ipv4.ping_group_range": "0 1",
```

Comment:
ICBW, but I _think_ this can be "0 0".

Reply:
I could not tell if this was starting and number versus start and end.

Reply:
I'm not super-pumped about using this if the documentation is this scarce, but it's a range of values - the GID of a user would need to be greater than or equal to the first value, and less than or equal to the second value.  The default of `[1 0]` chosen so that when left unconfigured, access can't be granted by this mechanism.